R/page_numbering.R

```diff
@@ -0,0 +1,112 @@
+
+#' @title Page Numbering
+#' 
+#' @description
+#' Class for numbering pages in a logical manner.
+PageNumbering <- R6::R6Class(
+  classname = "Page Numbering",
```

Comment:
This should not contain spaces.
See:
https://r6.r-lib.org/reference/R6Class.html
https://adv-r.hadley.nz/s3.html#s3-classes